plugin-java/che-plugin-java-ext-jdt/org-eclipse-jdt-ui/src/main/java/org/eclipse/jdt/internal/core/JavaModelManager.java

```diff
@@ -2865,9 +2865,9 @@ public String toString() {
     }
 
     public JavaWorkspaceScope getWorkspaceScope() {
-        if (this.workspaceScope == null) {
+//        if (this.workspaceScope == null) {
```

Comment:
no, it's eclipse code, https://jira.codenvycorp.com/browse/IDEX-3609 after this I return commented code